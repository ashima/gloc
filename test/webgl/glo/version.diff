--- version.true	2011-08-22 17:43:13.000000000 -0700
+++ version.out	2011-10-28 14:07:40.000000000 -0700
@@ -1 +1,22 @@
-error
+{
+  "glo": [ 0, 8, 0 ],
+  "target": [ "webgl", [ 1, 0, 0 ] ],
+  "meta": {
+    "author": {},
+    "license": [ "Copyright (C) 2011 . All rights reserved.", "" ],
+    "library": null,
+    "version": null,
+    "build": null
+  },
+  "units": [
+    {
+      "insym": [ "GLOC_VERSION_100" ],
+      "outsym": [],
+      "inmac": [],
+      "opmac": [],
+      "outmac": [],
+      "source": "#line 3 GLOC_0\nvoid main() { }"
+    }
+  ],
+  "linkmap": {}
+}
