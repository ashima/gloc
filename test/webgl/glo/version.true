{
  "glo": [ 0, 1, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "copyright": [ "", 2011, "" ],
    "author": {},
    "license": null,
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [ "GLOC_VERSION_100" ],
      "outsym": [ "main" ],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source": "#line 3 GLOC_0\nvoid main() { }"
    }
  ],
  "linkmap": { "0": "<stdin>#0" }
}
