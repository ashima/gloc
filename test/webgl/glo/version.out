{
  "glo": [ 0, 8, 0 ],
  "target": [ "webgl", [ 1, 0, 0 ] ],
  "meta": {
    "author": {},
    "license": [ "Copyright (C) 2011 . All rights reserved.", "" ],
    "library": null,
    "version": null,
    "build": null
  },
  "units": [
    {
      "insym": [],
      "outsym": [],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source": ""
    },
    {
      "insym": [],
      "outsym": [],
      "inmac": [],
      "opmac": [],
      "outmac": [],
      "source": "\n#line 1 0\n#version 100\n\nvoid main() { }"
    }
  ],
  "linkmap": {}
}
